eq-author/src/App/QuestionnaireDesignPage/index.js

```diff
@@ -81,31 +86,48 @@ export class UnwrappedQuestionnaireDesignPage extends Component {
     } = this.props;
     const { entityName, entityId } = match.params;
 
-    let sectionId, position;
-    if (entityName === SECTION) {
-      sectionId = entityId;
-      position = 0;
-    } else if (entityName === PAGE) {
-      for (let i = 0; i < questionnaire.sections.length; ++i) {
-        const section = questionnaire.sections[i];
-        const page = find(section.pages, { id: entityId });
-        if (page) {
-          sectionId = section.id;
-          position = section.pages.indexOf(page) + 1;
-          break;
+    let sectionId, position, selectedPage, selectedSection, selectedFolder;
+
+    switch (entityName) {
+      case SECTION:
+        sectionId = entityId;
+        position = 0;
+        break;
+
+      case QUESTION_CONFIRMATION:
```

Comment:
```suggestion
```

Love a good switch statement 'though. Nice one. 👌😉

Reply:
Haha, cheers Matt. That case is intentional though - it means that the following block handles both QUESTION_CONFIRMATION and PAGE types. I can add a "// Fall though" comment if you want so that it's clear that it is intentional?

Reply:
Oh right! Okay. No, that's fine.